examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
Depending on the potential type, in some cases the user will load them from some library (example openkim), or from some files (example, as tables, or as parameters in a special format). In the case of LJ, or simple analytic potentials, often the user will just define the parameters synamically. The idea to have some standard format for the potential files is exactly what OpenKIM.org projects does. Though currently it is limtted to metallic/ceramic systems. There is no standard for DEM or DPD potentials. In the DOW we proposed to consider the extension of OpenKIM. The option to allow the user to either directly create yml files for the potentials, or alternatively use a predefined library of SimPhoNy supported potentials is not bad, at least for the current state. We can always, later on add "methods" to define such files or objects dynamically.